### PR TITLE
Adls data plane task updateversion

### PR DESCRIPTION
**Pull request by rahuldutta90**
## Description
Update the version of the ADLS dataplane SDK to 1.1.2 

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/DataLakeStore/AzureRM.DataLakeStore.psd1 [hunk 1]
```diff
@@ -152,11 +152,7 @@ PrivateData = @{
 
         # ReleaseNotes of this module
         ReleaseNotes = '
-		* Export-AzureRmDataLakeStoreItem (https://github.com/Azure/azure-powershell/blob/adls-data-plane/src/ResourceManager/DataLakeStore/documentation/upcoming-breaking-changes.md) - Deprecated parameters PerFileThreadCount, ConcurrentFileCount and introduced parameter Concurrency
-		* Import-AzureRMDataLakeStoreItem (https://github.com/Azure/azure-powershell/blob/adls-data-plane/src/ResourceManager/DataLakeStore/documentation/upcoming-breaking-changes.md) -Deprecated parametersPerFileThreadCount, ConcurrentFileCount and introduced parameter Concurrency
-		* Get-AzureRMDataLakeStoreItemContent - Fixed the tail behavior for contents greater than 4MB
-		* Set-AzureRMDataLakeStoreItemExpiry - Introduced new parameter set SetRelativeExpiry for setting relative expiration time
-		* Remove-AzureRmDataLakeStoreItem (https://github.com/Azure/azure-powershell/blob/adls-data-plane/src/ResourceManager/DataLakeStore/documentation/upcoming-breaking-changes.md) - Deprecated parameter Clean.
+		* Update the version of the ADLS dataplane SDK to 1.1.2
```
Comment (cormacpayne) [inline src/ResourceManager/DataLakeStore/AzureRM.DataLakeStore.psd1]: @rahuldutta90 you can revert the change made to the `ReleaseNotes` -- we will update all of this information when we are ready to release this module again

Comment (rahuldutta90) [reply src/ResourceManager/DataLakeStore/AzureRM.DataLakeStore.psd1]: @cormacpayne Reverted it.

**Comment by maddieclayton** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/512/
